src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_DIMENSION = 10;
+    private static final int DEFAULT_FIGURE_RADIUS = 10;
+    private static final int MAX_FIGURES = 5;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int figureType = random.nextInt(MAX_FIGURES);
+        String color = colorSupplier.getRandomColor();
+        switch (figureType) {
+            case 0:
+                return createSquare();
+            case 1:
+                return createRightTriangle();
+            case 2:
+                return createRectangle();
+            case 3:
+                return createIsoscelesTrapezoid();
+            case 4:
+                return createCircle();
+            default:
+                return getDefaultFigure();
+        }
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEFAULT_FIGURE_RADIUS, Color.WHITE.name());
+    }
+
+    private Circle createCircle() {
+        int radius = (random.nextInt(MAX_DIMENSION) + 1) * MAX_DIMENSION;

```

Comment:
create separate method for `(random.nextInt(MAX_DIMENSION) + 1) * MAX_DIMENSION`